code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -125,7 +125,7 @@ func (nar *newAllocationRequest) validate(now time.Time, conf *Config) error {
 	}
 
 	dur := common.ToTime(nar.Expiration).Sub(now)
-	if dur < conf.MinAllocDuration {
+	if dur < conf.TimeUnit {
```

Comment:
This would make the allocation has min allocation duration of `720hours`, should we do this? I think we have system test cases that expect the allocation to be expired after the min duration of `5m`.

Reply:
@peterlimg we changed tests as well, @guruhubb confirmed that allocation duration will be not less than TimeUnit from now on